hooks/hookstage/processedauctionrequest.go

```diff
@@ -26,5 +26,5 @@ type ProcessedAuctionRequest interface {
 // ProcessedAuctionRequestPayload consists of the openrtb_ext.RequestWrapper object.
 // Hooks are allowed to modify openrtb_ext.RequestWrapper using mutations.
 type ProcessedAuctionRequestPayload struct {
-	RequestWrapper *openrtb_ext.RequestWrapper
+	BidRequest *openrtb_ext.RequestWrapper
```

Comment:
I think naming `BidRequest` to `request wrapper` object will be misleading as `openrtb_ext.RequestWrapper` also have bidRequest object embedded
What are your thoughts?

Reply:
I think I prefer it being named `RequestWrapper` since when we actually go ahead and access the BidRequest within the wrapper, right now it looks like `payload.BidRequest.BidRequest`, and I think it's clearer for me to read something like `payload.RequestWrapper.BidRequest`. I'm also fine with it being `payload.Request.BidRequest`

Reply:
Yep `Request` will be fine

Reply:
Done